benefits/core/migrations/0002_data.py

```diff
@@ -188,7 +188,7 @@ def load_data(app, *args, **kwargs):
         jwe_encryption_alg=os.environ.get("COURTESY_CARD_VERIFIER_JWE_ENCRYPTION_ALG", "RSA-OAEP"),
         jws_signing_alg=os.environ.get("COURTESY_CARD_VERIFIER_JWS_SIGNING_ALG", "RS256"),
         auth_provider=None,
-        selection_label_template="core/includes/selection_label__mst_courtesy_card.html",
+        selection_label_template="core/includes/selection-label--courtesy-card.html",
```

Comment:
I actually think this should keep the `mst` moniker, as it is their `courtesy_card` program.

Reply:
Made the filename `selection-label--mst-courtesy-card.html` in [95d85ca](https://github.com/cal-itp/benefits/pull/1513/commits/95d85cadc21a692d83d09e425032d3f9b4cfcb15)